include/zephyr/drivers/uart.h

```diff
@@ -561,10 +561,11 @@ static inline int z_impl_uart_poll_in_u16(const struct device *dev,
 /**
  * @brief Write a character to the device for output.
  *
- * This routine checks if the transmitter is full.  When the
+ * This routine checks if the transmitter is full. When the
  * transmitter is not full, it writes a character to the data
- * register. It waits and blocks the calling thread, otherwise. This
- * function is a blocking call.
+ * register. It waits and blocks the calling thread otherwise. This
+ * function is a blocking call. It blocks the calling thread until the
+ * character is sent.
```

Comment:
Doesn't it block until the TX FIFO has space for the character, rather than until the character is sent?

Reply:
The doc I linked says "until the give character is sent". My intention is to unify these docs.